include/alp/amf-based/vector.hpp

```diff
@@ -58,6 +58,15 @@ namespace alp {
 		void setInitialized( alp::Vector< T, Structure, Density::Dense, View, ImfR, ImfC, backend > & v, bool initialized ) noexcept {
 			setInitialized( static_cast< typename alp::Vector< T, Structure, Density::Dense, View, ImfR, ImfC, backend >::base_type &>( v ), initialized );
 		}
+
+		template< typename T, typename Structure, typename View, typename ImfR, typename ImfC, enum Backend backend >
```

Comment:
Template specifications should be at most 80 lines long, can you please break this into lines. Also applies to other locations.